app/src/main/java/org/mozilla/reference/browser/components/Services.kt

```diff
@@ -0,0 +1,37 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.reference.browser.components
+
+import android.content.Context
+import mozilla.components.browser.storage.sync.PlacesHistoryStorage
+import mozilla.components.browser.storage.sync.SyncAuthInfo
+import mozilla.components.feature.sync.FirefoxSyncFeature
+import mozilla.components.feature.tabs.TabsUseCases
+import org.mozilla.reference.browser.browser.FirefoxAccountsIntegration
+
+/**
+ * Component group for all application services.
+ */
+class Services(
+    private val context: Context,
+    private val tabsUseCases: TabsUseCases,
+    private val placesHistoryStorage: PlacesHistoryStorage
+) {
+    /**
+     * Feature component to integrate with Firefox Sync.
+     */
+    val sync by lazy {
+        FirefoxSyncFeature(mapOf("history" to placesHistoryStorage)) {
+            SyncAuthInfo(it.kid, it.fxaAccessToken, it.syncKey, it.tokenServerUrl)
+        }
+    }
+
+    /**
+     * Integration component for Firefox Accounts.
+     */
+    val accounts: FirefoxAccountsIntegration by lazy {
```

Comment:
Won't an `account` service (singular) read better in code? There's only one (or none) account, and the service provides account to a single account type, and there's a single account service... etc., all singular.

Reply:
Yeah, I was thinking about that, but we use plural everywhere: https://fxa.readthedocs.io/en/latest/

FxA always stands for "Firefox Accounts" and the Firefox Accounts servers etc.